docusaurus/docs/operate/run_a_node/full_node_cosmovisor.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Full Node - Cosmovisor
+sidebar_position: 1
+---
+
+# Full Node - Docker <!-- omit in toc -->
+
+- [What is a Full Node](#what-is-a-full-node)
+- [What is Cosmovisor](#what-is-cosmovisor)
+
+## What is a Full Node
```

Comment:
You already put together the documantion in [1]. Why are we replicating the full node details?

[1] https://dev.poktroll.com/operate/run_a_node/full_node

Reply:
@Olshansk I was thinking of showing how to use different tools. In addition to docker-compose, one can use cosmovisor.